library/src/main/java/com/novoda/downloadmanager/FileCallbackThrottle.java

```diff
@@ -0,0 +1,32 @@
+package com.novoda.downloadmanager;
+
+/**
+ * Controls the rate at which a {@link DownloadFile} calls back to
+ * a {@link DownloadBatch}. Essentially controls the rate of notifications
+ * during a download of a file.
+ * <p>
+ * Note: This does not control the overall emissions of {@link DownloadBatchStatus} between the client and this library.
+ */
+public interface FileCallbackThrottle {
```

Comment:
Sorry I'm late I just managed to check this PR, but what about adding a link in this class' Javadoc to the method the client can use to specify a custom Throttle?

Reply:
Sure thing, here you go 😄 https://github.com/novoda/download-manager/blob/v2_java_doc/library/src/main/java/com/novoda/downloadmanager/FileCallbackThrottle.java#L6